test/clj/rems/api/test_api.clj

```diff
@@ -19,8 +19,8 @@
         (is (response-is-not-found? resp))))
     (testing "authorized,"
       (testing "missing params"
-        ;; Surprisingly hard to find a POST route that isn't shadowed
-        ;; by a GET route. For example, GET /api/applications/command
+        ;; surprisingly hard to find a post route that isn't shadowed
```

Comment:
did you lowercase the whole file by accident? :smile:

Reply:
yeah